exporter/importables.go

```diff
@@ -174,7 +174,7 @@ var resourcesMap map[string]importable = map[string]importable{
 				ic.Emit(&resource{
 					Resource: "databricks_permissions",
 					ID:       fmt.Sprintf("/instance-pools/%s", r.ID),
-					Name:     "inst_pool_" + ic.Importables["databricks_instance_pool"].Name(ic, r.Data),
+					Name:     "inst-pool_" + ic.Importables["databricks_instance_pool"].Name(ic, r.Data),
```

Comment:
although `-` is allowed in resources identifiers, they aren't considered idiomatic: https://github.com/hashicorp/hcl/blob/main/hclsyntax/spec.md#identifiers